docs/Configuring-Tachyon-with-Swift.md

```diff
@@ -28,67 +28,34 @@ To configure Tachyon to use Swift as its under storage system, modifications to
 storage system address. You specify it by modifying `conf/tachyon-env.sh` to include:
 
 ```bash
-export TACHYON_UNDERFS_ADDRESS=swift://containter.swift1
+export TACHYON_UNDERFS_ADDRESS=swift://<swift-containter>
 ```
 
-Where `container` is an existing Swift container and `swift1` is a profile in `core-site.xml`. By
-default, Tachyon uses the `conf` directory to load `core-site.xml`. To specify a different location,
-please configure `tachyon.underfs.hadoop.configuration` accordingly.
+Where `<swift-container>` is an existing Swift container.
 
-Swift depends on Hadoop version 2.3.0 or later and can be configured via `hadoop-openstack.version`
-in the main pom.xml. Make sure to compile Tachyon with the same Hadoop version as
-`hadoop-openstack.version`. For example, to compile Tachyon with Swift and Hadoop 2.4.0 use:
+The following configuration should be provided in the `conf/tachyon-env.sh`
 
-```bash
-$ mvn -Dhadoop.version=2.4.0 clean package
-```
-
-# Configuring Hadoop
-
-After the build is successful, the `core-site.xml` configuration file needs to be changed. The
-configuration template can be found in the `conf/core-site.xml.template` file; it
-contains three example sections: local Swift based on Keystone authentication model, local Swift
-based on temp authentication model, and SoftLayer public object store. The general structure of the
-parameters is `fs.swift.service.<PROFILE>.<PARAMETER>` where `<PROFILE>` is a name that will be
-later used as a part of the Swift URL. For example, if `<PROFILE>` is “swift1” then the Swift URL
-would be
-
-	swift://<SWIFT CONTAINER>.swift1/
-
-Edit `core-site.xml` and update `fs.swift.service.<PROFILE>.auth.url`.
-
-For Swift using Temp Auth or SoftLayer, update:
 
-	fs.swift.service.<PROFILE>.apikey, fs.swift.service.<PROFILE>.username
+ 	-Dfs.swift.user=<swift-user>
+  	-Dfs.swift.tenant=<swift-tenant>
+  	-Dfs.swift.apikey=<swift-user-password>
+  	-Dfs.swift.auth.url=<swift-auth-url>
+  	-Dfs.swift.auth.port=<swift-auth-url-port>
+  	-Dfs.swift.use.public.url=<swift-use-public>
+  	-Dfs.swift.auth.method=<swift-auth-model>
+  	
+Possible values of `<swift-use-public>` are `true`, `false`.
+Possible values of `<swift-auth-model>` are `keystone`,
+`tempauth`, `swiftauth`
 
-For Swift using Keystone Auth, update:
+On the successfull authentication, Keystone will return two access URLs: public and private. If Tachyon is used inside company network and Swift located on the same network it's adviced to set value of `<swift-auth-model>`  to `false`.
```

Comment:
If Tachyon is used inside company network and Swift `is` located on the same network it is `advised` to set value of ...